### PR TITLE
Fixed validation error 

### PR DESCRIPTION
**Pull request by pontushojer**
Fix for issue #85. Reference_variants defaults to None which fails validation (string required). Also updated this for other paths in config.schema.yaml and config.yaml.

### CODE REVIEW

File: config.schema.yaml [hunk 1]
```diff
@@ -40,8 +40,8 @@ properties:
...
     type: string
     description: Path to reference variants, if not provided then variant will be called by freebayes
-    default:
+    default: ""
   phasing_ground_truth:
     type: string
     description: Path to phased variants used to compute phasing stats.
-    default: .
\ No newline at end of file
+    default: ""
```
Comment (marcelm) [inline config.schema.yaml]: There’s a missing newline at the end of the file

Comment (pontushojer) [reply config.schema.yaml]: Fixed in new commit.

**Comment by marcelm** [comment]
I would suggest to fix the problem in another way: By changing the schema such that it allows `null` in addition to a `string` for the affected fields. According to https://stackoverflow.com/questions/16241333/ , you can use `type: "string, null"` to achieve this.

That way, one can just leave out the value (as your intention was from the beginning I gather) instead of having to specify an empty string with `""`.

**Comment by pontushojer** [comment]
> I would suggest to fix the problem in another way: By changing the schema such that it allows `null` in addition to a `string` for the affected fields. According to https://stackoverflow.com/questions/16241333/ , you can use `"type": ["string", "null"]` or perhaps `"type": "string, null"` to achieve this.
> 
> That way, one can just leave out the value (as your intention was from the beginning I gather) instead of having to specify an empty string with `""`.
> 
> _Edit:_ Fixed suggested schema.

I did not know that was an option but that is indeed much nicer. Yes my intension was to just leave it out the value completely. 

Changed in new commit.

**Pull request merged**